Gifski/TrimmingAVPlayerView.swift

```diff
@@ -0,0 +1,47 @@
+import AVKit
+
+final class TrimmingAVPlayerView: AVPlayerView {
+	private var timeRangeObserver: NSKeyValueObservation?
+
+	func observeTrimmedTimeRange(_ updateClosure: @escaping (ClosedRange<Double>) -> Void) {
+		timeRangeObserver = player?.currentItem?.observe(\.duration, options: [.new]) { item, _ in
```

Comment:
`.duration` doesn't work on Mojave. It's never called, but `.forwardPlaybackEndTime` works fine. It's triggered when both changing the start and end trim. Can you check if that works on Catalina? If so, just add a note to switch to `.duration` when we target macOS 10.15.